launch/sim_iarc7_safety.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
We can have parameters internal to safety that get loaded from a yaml file, and we can even have an argument that indicates some mode which internally loads a different yaml file